plugins/generator-1.19.4/forge-1.19.4/mappings/gamerules.yaml

```diff
@@ -0,0 +1,48 @@
+_default: null
+_mcreator_prefix: "CUSTOM:"
+_mcreator_map_template: "@JavaModNameGameRules.@REGISTRYNAME"
+ANNOUNCE_ADVANCEMENTS: GameRules.RULE_ANNOUNCE_ADVANCEMENTS
+COMMAND_BLOCK_OUTPUT: GameRules.RULE_COMMANDBLOCKOUTPUT
+DISABLE_ELYTRA_MOVEMENT_CHECK: GameRules.RULE_DISABLE_ELYTRA_MOVEMENT_CHECK
+DISABLE_RAIDS: GameRules.RULE_DISABLE_RAIDS
+DO_DAYLIGHT_CYCLE: GameRules.RULE_DAYLIGHT
+DO_ENTITY_DROPS: GameRules.RULE_DOENTITYDROPS
+DO_FIRE_TICK: GameRules.RULE_DOFIRETICK
+DO_INSOMNIA: GameRules.RULE_DOINSOMNIA
+DO_IMMEDIATE_RESPAWN: GameRules.RULE_DO_IMMEDIATE_RESPAWN
+DO_LIMITED_CRAFTING: GameRules.RULE_LIMITED_CRAFTING
+DO_MOB_LOOT: GameRules.RULE_DOMOBLOOT
+DO_MOB_SPAWNING: GameRules.RULE_DOMOBSPAWNING
+DO_TILE_DROPS: GameRules.RULE_DOBLOCKDROPS
+DO_WEATHER_CYCLE: GameRules.RULE_WEATHER_CYCLE
+DROWNING_DAMAGE: GameRules.RULE_DROWNING_DAMAGE
+FALL_DAMAGE: GameRules.RULE_FALL_DAMAGE
+FIRE_DAMAGE: GameRules.RULE_FIRE_DAMAGE
+KEEP_INVENTORY: GameRules.RULE_KEEPINVENTORY
+LOG_ADMIN_COMMANDS: GameRules.RULE_LOGADMINCOMMANDS
+MOB_GRIEFING: GameRules.RULE_MOBGRIEFING
+NATURAL_REGENERATION: GameRules.RULE_NATURAL_REGENERATION
+REDUCED_DEBUG_INFO: GameRules.RULE_REDUCEDDEBUGINFO
+SEND_COMMAND_FEEDBACK: GameRules.RULE_SENDCOMMANDFEEDBACK
+SHOW_DEATH_MESSAGES: GameRules.RULE_SHOWDEATHMESSAGES
+SPECTATORS_GENERATE_CHUNKS: GameRules.RULE_SPECTATORSGENERATECHUNKS
+MAX_COMMAND_CHAIN_LENGTH: GameRules.RULE_MAX_COMMAND_CHAIN_LENGTH
+MAX_ENTITY_CRAMMING: GameRules.RULE_MAX_ENTITY_CRAMMING
+RANDOM_TICK_SPEED: GameRules.RULE_RANDOMTICKING
+SPAWN_RADIUS: GameRules.RULE_SPAWN_RADIUS
+DO_PATROL_SPAWNING: GameRules.RULE_DO_PATROL_SPAWNING
+DO_TRADER_SPAWNING: GameRules.RULE_DO_TRADER_SPAWNING
+FORGIVE_DEAD_PLAYERS: GameRules.RULE_FORGIVE_DEAD_PLAYERS
+UNIVERSAL_ANGER: GameRules.RULE_UNIVERSAL_ANGER
+PLAYERS_SLEEPING_PERCENTAGE: GameRules.RULE_PLAYERS_SLEEPING_PERCENTAGE
+FREEZE_DAMAGE: GameRules.RULE_FREEZE_DAMAGE
+DO_WARDEN_SPAWNING: GameRules.RULE_DO_WARDEN_SPAWNING
+BLOCK_EXPLOSION_DROP_DECAY: GameRules.BLOCK_EXPLOSION_DROP_DECAY
```

Comment:
Those are not named like this in Minecraft.

If you tested this PR with new additions you added, you would notice this.

Reply:
Sorry for trusting @Goldorion with the mappings

Reply:
Well if you copy something to PR, it is on you to stand behind changes.

CLA of PR is signed by PR author so PR is at the end of the day always work of the PR author directly or indirectly

Reply:
I have found out a bug I introduced in gamerules and I will fix mappings at the same time

Reply:
This is fixed, I went into MC's source code to check game rules names